docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -27,7 +27,7 @@ To start, open a terminal and set the `OfficeWebAddinDeveloperExtras` property f
 
 Then, open the Office application and [sideload your add-in](sideload-an-office-add-in-on-ipad-and-mac.md). Right-click the add-in and you should see an **Inspect Element** option in the context menu.  Select that option and it will pop the Inspector, where you can set breakpoints and debug your add-in.
 > [!NOTE]
-> If you are trying to use the inspector and the dialog flickers, try the following workaround:
+> If you are trying to use the inspector and the dialog flickers, try updating your version of Office. If that doesn't resolve the flicker, try the following workaround:
```

Comment:
Consider changing "try updating your version of Office." -> "update Office to the latest version."

Reply:
+1

Reply:
+1